src/workspaces/SettingsModal/utils.ts

```diff
@@ -18,8 +19,9 @@ export type {
 
 export const suggestedPrefixes = {
   allObjects: 'All Objects',
-  submissions: 'submissions/',
   submissionIntermediaries: 'submissions/intermediates/',
+  submissionFinalOutputs: 'submissions/final-outputs/',
```

Comment:
Added "final-outputs" and reordered slightly to reflect what we think folks are likely to select.